omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", git: "https://github.com/chef/omnibus"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software"
+gem "omnibus", git: "https://github.com/chef/omnibus", branch: "lcg/improve-appbundling"
+gem "omnibus-software", git: "https://github.com/chef/omnibus-software", branch: "lcg/appbundle-0.11-without"
```

Comment:
i suspect that master of both with appbundler 0.11.0 will work fine, since using `without` doesn't DTRT anyway right now...